backend/db/resolve_fetch_failures_0_members.py

```diff
@@ -93,23 +95,36 @@ def resolve_fetch_failures_0_members(
         # Fetch data
         csets_and_members: Dict[str, List[Dict]] = fetch_cset_and_member_objects(
             codeset_ids=list(failed_cset_ids), flag_issues=False)
-        # - filter success & track results
-        csets_and_members['OMOPConceptSet'] = [
-            x for x in csets_and_members['OMOPConceptSet'] if x['member_items']]
-        cset_is_draft_map.update({x['properties']['codesetId']: x['properties']['isDraft'] for x in csets_and_members['OMOPConceptSet']})
-        success_cases: List[int] = [
-            x['properties']['codesetId'] for x in csets_and_members['OMOPConceptSet']]
-        failed_cset_ids = list(set(failed_cset_ids) - set(success_cases))
+
+        # - identify discarded drafts
+        # TODO: ascertain discarded drafts
+        #  - how to ascertain?
+        #    - 1 (ok~): detect discarded drafts as not returned from fetch_cset_and_member_objects(). ~confirm w/ amin et al
+        #    - 2 (x): Ask Amin et al: is the enclave API reliable for discarded drafts? Some of these are appearing in the
+        #    enclave API but not in the UI. x: I think (2) is not a case I need to worry about. i think they weren't discared.
+        #    - This line below should be enough to handle. Just want to verify (1)
+        discarded_cset_ids: Set[int] = set([x for x in failed_cset_ids if x not in csets_and_members['OMOPConceptSet']])
+        # TODO: mark discarded drafts resolved
+        # _report_success(discarded_cset_ids, failure_lookup, 'Success result: Discarded draft', use_local_db)
+
+        # - identify persistent drafts
```

Comment:
## How to handle persistent drafts?
I think I'll make an issue for this.

I think there are drafts that will remain drafts permanently. With the current setup, we're just going to keep trying to fetch their members every 20 minutes forever.

Perhaps I can update `fetch_audit` with a new field `last_fetch_attempt`, and for drafts that are old (1 day; 1 week?) we check less often? How often? 1x/day? It will be annoying though for someone who marks their draft resolved after a few weeks and wants to see it in TermHub immediately. Perhaps the way around this is if there's some way for the refresh GH action to know if it was triggered manually, rather than on cronjob, then it should automatically trigger the `fetch_failures_0_members` for these persistent drafts as well.

Reply:
I created an issue:
- #759